types/v56/vm_types.go

```diff
@@ -107,3 +110,35 @@ type ComputePolicy struct {
 	VmSizingPolicy         *Reference `xml:"VmSizingPolicy,omitempty"`         // VdcComputePolicy that defines VM's sizing and resource allocation.
 	VmSizingPolicyFinal    *bool      `xml:"VmSizingPolicyFinal,omitempty"`    // True indicates that the sizing policy cannot be removed from a VM that is instantiated with it. This value defaults to false.
 }
+
+type CreateVmParams struct {
```

Comment:
Related to the above comment. In general, it would be very useful to have comments explaining which types are used for standalone VM and which for normal. It is not visible for someone who does not know the logic of how they are differentiated.

Reply:
done